src/pages/install/macos.md

```diff
@@ -0,0 +1,34 @@
+
+# Installing RGBDS on macOS
+
+RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
+
+```bash
+brew install rgbds
+```
+
+If not, and you want to install version v0.5.2 or newer, you can download our pre-built binaries.
+
+1. Select [the version you want](/docs/).
+2. Under "GitHub links", follow the "release page" link.
+3. Scroll down to "Assets".
+4. Download `rgbds-<version>-macos-x86-64.zip` (so, for example, `rgbds-0.7.0-macos-x86-64.zip`)
```

Comment:
```suggestion
4. Download <code>rgbds-<i>&lt;version&gt;</i>-macos-x86-64.zip</code> (for example, version 0.7.0 would have `rgbds-0.7.0-macos-x86-64.zip`).
```

Reply:
Honestly I still think these are just way too many steps just to get the download. Why not just have a "Downloads" page with links to each binary release in the main website? (Not too different from what SameBoy's website does)

Reply:
The authoritative list is GitHub's Releases section, so any list on this website would have to be maintained separately... :/

Reply:
Sure, you only publish a release that often. Even just having links to the latest binaries on the front page or something would be nice, instead of having to go install -> docs -> github -> correct file.

Reply:
It still adds another manual edit to potentially forget, in a process that could be a year+ since we last did it.

Reply:
Well, since the docs are updated automatically via CI, it should be possible to keep a list of the versions with static bins in a JSON file, and `import` it for processing in those MDX files...

I've just not fleshed that out, since we'd need to know which artifacts are present on the release page... but I think it *should* be possible. And certainly worth the effort to save two of the clicks Liji pointed out.

Reply:
If it can be done automatically, yeah, that's worthwhile. But this PR is focused on rephrasing the docs for the current process, so let's call that a separate issue.

Reply:
Allow me to disagree: such a list would drastically alter the way the instructions are presented. And, further, the "backend" JSON would have to be manually written to "seed" it, so might as well in this PR.

Reply:
I agree with @Rangi42 - This is out of scope here and requires additional that have little to do with the rephrasing we are doing here - let's open a separate issue for this.

Reply:
Yeah, it turns out to be more than anticipated, and so has the "base" PR. I agree.